src/modules/terraformPlan/repo/DynamoDbMetadataRepo.ts

```diff
@@ -30,10 +28,11 @@ export class DynamoDBMetadataRepo implements IMetadataRepository {
     stack: string,
     commitSHA: string
   ): Promise<TerraformPlan> {
-    const params: ScanCommandInput = {
+    const params: QueryCommandInput = {
       TableName: this.tableName,
+      KeyConditionExpression: "#commitSHA= :commitSHA",
       FilterExpression:
-        "#owner = :owner and #repo = :repo and #commitSHA = :commitSHA and #component = :component and #stack = :stack",
+          "#owner = :owner and #repo = :repo and #component = :component and #stack = :stack",
```

Comment:
@mcalhoun because we filter sha with KeyCondition 
https://github.com/cloudposse/github-action-terraform-plan-storage/pull/31/files#diff-95366d8215594fbafa600c4d01acd3040d43de241933dcf7a6ad86ead80e391aR33
```
KeyConditionExpression: "#commitSHA= :commitSHA",
```